canvasapi/paginated_list.py

```diff
@@ -57,8 +57,23 @@ def _get_next_page(self):
         )
         data = response.json()
         self._next_url = None
+        # Check the response headers first. This is the normal Canvas convention
+        # for pagination, but there are endpoints which return a `meta` property
+        # for pagination instead.
+        # See https://github.com/ucfopen/canvasapi/discussions/605
+        if response.links:
+            next_link = response.links.get("next")
+        elif type(data) is dict and data.get("meta") is not None:
```

Comment:
This is such a strange case in the API, but from what I can tell, `pagination` will always be within `meta`, so checking that it exists should be enough for this condition and then the `KeyError` exception would stop the iterator of the `next` key is missing.

All that said, since this seems to be the only endpoint which uses this pagination method, is it worth just making the `LiveAssessment` class iterable?